cmd ref: describe `update available` output (for import stages) in `status` #477 (#1299)
iterative/dvc.org: content/docs/command-reference/status.md

    hash in the DVC-file is up to date, but there is no corresponding
    <abbr>cache</abbr> file or directory.

> - _update available_ means that <abbr>import stages</abbr> are not updated to
>   latest version. The orginal file or directory from where the import was done
>   is changed. The imported file can me moved to latest version by running
>   `dvc update`.

Reviewer: ```suggestion
- _update available_ means that <abbr>import stages</abbr> are outdated.
  The original file or directory imported has changed. The imported data can be
  moved to its latest version by using `dvc update`.
```

My suggestion probably needs rewrapping (80 char formatting).
> You can use an IDE plugin, and the pre-commit hook should enforce this, as you know (explained in our [docs contrib guide](https://dvc.org/doc/user-guide/contributing/docs#doc-style-guidelines-javascript-and-markdown)).

Please also install a spell checker to your IDE (typo: "orginal"). I use **Code Spell Checker** for VSCode.

The output shows where the location of the remote storage is, as well as any
differences between the <abbr>cache</abbr> and `storage` remote.

> ## Example: Import Stage

Reviewer: ```suggestion
## Example: Import stage
```

> Let us import a file `data.csv` from a <abbr>DVC repository</abbr> stored
> locally to our current repository using `dvc import`.
> 
> ```dvc
> $ dvc import /home/user/path/ data.csv
> Importing 'data.csv (/home/user/path/)'
> -> 'data.csv'

Reviewer: Nice. Let me just suggest slightly simpler language:
```suggestion
Let's import a data file (`data.csv`) from a different <abbr>DVC
repository</abbr> into our current project using `dvc import`:

```dvc
$ dvc import different/repo/location data.csv
```

Also note `dvc import` accepts both paths and URLs so I just generalized the example even more, to `different/repo/location`.

We also try to avoid command output unless its totally necessary (e.g. if this was the cmd ref of `dvc import` itself) because this output tends to change a lot since DVC core is in active development so the examples are easier to maintain like this. thanks for actually running the command though!

```

> This is called a <abbr>import stage</abbr>. If the orginal file or directory is
> changed then running `dvc status` will show `update available` as output.

Reviewer: ```suggestion
The resulting `data.csv.dvc` file is called an <abbr>import stage</abbr>. If the
original file or directory changes later, `dvc status` will show `update available`
as output:
```
> Please check paragraph wrapping.

(Typo: "orginal")

```dvc
$ dvc status
data.csv.dvc:
	changed deps:
		update available:   data.csv (/home/user/path/)
```

> This shows that orginal file or directory is changed. The import stage can be
> brought to lates version by using `dvc update`.

Reviewer: ```suggestion
The imported data can be brought to its latest version by using `dvc update`.
```
that's good enough since the previous paragraph already explained about the source being changed.

But definitely spell checker please (typos: "orginal", "lates") 🙂

Author: Thanks for your inputs and plugin suggestion as I use VSCode too. 
I changed the location to `different/repo/location` in `dvc status` output also.
